### PR TITLE
[DE] Add `Client/Options/Universal_offset`, `Client/Options/Offset_Wizard` and `Guides/How_to_use_the_offset_wizard`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Guides/How_to_Use_the_Offset_Wizard/de.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+Wenn die [Hitsounds](/wiki/Beatmapping/Hitsound) in jeder gespielten Beatmap nicht mit der Musik synchron sind, ist es sehr wahrscheinlich, dass du dein [universales Offset](/wiki/Client/Options/Universal_offset) umstellen musst. Am einfachsten geht das mit dem **Offset-Assistenten**.
+
+Führe die folgenden Schritte aus, um dein universales Offset mit Hilfe des Offset-Assistenten anzupassen.
+
+## Den Offset-Assistenten öffnen
+
+1. Öffne osu!.
+2. Gehe zum Menü `Optionen` oder drücke `Strg` + `O`.
+3. Suche nach `offset` in der Schnellsuche und klicke auf den Button `Offset-Assistent starten`.
```
Comment (FamousPig) [inline wiki/Guides/How_to_Use_the_Offset_Wizard/de.md:13]: just a note. the English version uses double quotes here ("offset") but the inline code notation in the other article, so it's probably best to leave it.

**Review by FamousPig — APPROVED** [review]
👍

File: wiki/Client/Options/Offset_Wizard/de.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+Das aktuelle universale Offset wird in der linken oberen Ecke des Bildschirms angezeigt. Es kann über das Drücken der Tasten `Pfeil hoch` und `Pfeil runter` oder durch die Verwendung des Mausrads verändert werden.
+
+Unterhalb der Anzeige für das universale Offset befinden sich drei Optionen, die die Geschwindigkeit des Metronoms ändern. Durch die Aktivierung von `Halber Tick` wird die Zeitspanne zwischen den Ticks des Metronoms verdoppelt, wodurch es langsamer wird. Das Auswählen dieser Option wird ebenfalls die Option `Tick auf anderen Beat` anzeigen. Durch die Aktivierung von `Tick auf anderen Beat` wird die Zeitspanne zwischen den Ticks des Metronoms genauso lang wie bei `Halber Tick`, allerdings auf einem anderen Beat. Zuletzt wird die Aktivierung von `Doppelter Tick` die Zeitspanne zwischen den Ticks des Metronoms halbieren, wodurch es schneller wird. Diese Optionen werden verwendet, um das Metronom besser an den aktuell spielenden Song anzupassen.
```
Comment (respektive) [inline wiki/Client/Options/Offset_Wizard/de.md:13]:
```suggestion
Unterhalb der Anzeige für das universale Offset befinden sich drei Optionen, die die Geschwindigkeit des Metronoms ändern. Durch die Aktivierung von `Halber Tick` wird die Zeitspanne zwischen den Ticks des Metronoms verdoppelt, wodurch es langsamer wird. Das Auswählen dieser Option wird ebenfalls die Option `Tick auf anderen Beat` anzeigen. Durch die Aktivierung von `Tick auf anderen Beat` bleibt die Zeitspanne zwischen den Ticks des Metronoms genauso lang wie bei `Halber Tick`, allerdings auf einem anderen Beat. Zuletzt wird die Aktivierung von `Doppelter Tick` die Zeitspanne zwischen den Ticks des Metronoms halbieren, wodurch es schneller wird. Diese Optionen werden verwendet, um das Metronom besser an den aktuell spielenden Song anzupassen.
```

**Review by respektive — CHANGES_REQUESTED** [review]

File: wiki/Guides/How_to_Use_the_Offset_Wizard/de.md (hunk 1 above)
Comment (The-Last-Cookie) [reply wiki/Guides/How_to_Use_the_Offset_Wizard/de.md:13]: Yes, I was aware of that and wanted to keep it consistent with the other articles.

**Comment by The-Last-Cookie** [comment]
Thanks for the reviews!

**Review by respektive — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**